apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -434,4 +442,22 @@ defmodule OMG.Watcher.ExitProcessor do
     {:ok, spend_blknum} = position |> Utxo.Position.to_db_key() |> OMG.DB.spent_blknum()
     spend_blknum
   end
+
+  defp collect_invalidities_and_state_db_updates(
+         {ife_id, {input_exits, output_exits}},
+         {invalidities_by_ife_id, state_db_updates}
+       ) do
+    {:ok, input_exits_state_updates, {_, input_invalidities}} = State.exit_utxos(input_exits)
```

Comment:
leave also a comment here
```
# we can't call `State.exit_utxos(input_exits ++ output_exits)` because the types of these enumerable items are distinct
```

Reply:
Done